apps/labs/posts/numpy_benchmarking_blog.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 'NumPy Benchmarking'
+published: November 03, 2021
+author: khushi-agrawal
+description: 'My work was majorly focused on providing performance benchmarks to NumPy in realistic situations. The target was to show the world that NumPy is efficient in handling quasi real-life situations too.'
+category: [Developer workflows]
+featuredImage:
+  src: /posts/numpy_benchmarking_blog/feature.png
+  alt: 'A word cloud with themes, open-source projects and people mentioned throughout the blog post. Each is stylized using a different font, most of them calligraphical.'
+hero:
+  imageSrc: /posts/numpy_benchmarking_blog/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+In this blog post, I'll be talking about my journey in Quansight.
+I want to share all things I was involved in and accomplished.
+What issues I faced, and most importantly, what were awesome life hacks I learned during this period.
+
+First of all, I'd like to express my gratitude to the whole team
+for allowing me to be a part of such a great team.
+My work was majorly focused on providing performance benchmarks to NumPy in realistic situations.
+The target was to show the world that NumPy is efficient in handling quasi real-life situations too.
+
+The primary technical outcome of my work is available in the [numpy documentation](https://deploy-preview-461--numpy-preview.netlify.app/benchmark/).
+
+<p align="center">
+      <img src = "/posts/numpy_benchmarking_blog/journey.jpeg" alt = "A word cloud with themes, open-source projects and people mentioned throughout the blog post. Each is stylized using a different font, most of them calligraphical.">
```

Comment:
```suggestion
      <img src = "/posts/numpy_benchmarking_blog/journey.jpeg" alt = "A word cloud with themes, open-source projects and people mentioned throughout the blog post. Each is stylized using a different font, most of them calligraphical." />
```
Address recent [Vercel build error](https://vercel.com/quansight/labs/4Zpyfp4StihpT12WbJ5fQ1cJ6C4R), `Unexpected closing tag `</p>`, expected corresponding closing tag for `<img>``